src/runtime/virtcontainers/qemu.go

```diff
@@ -367,14 +372,34 @@ func (q *qemu) createQmpSocket() ([]govmmQemu.QMPSocket, error) {
 		path: monitorSockPath,
 	}
 
-	return []govmmQemu.QMPSocket{
+	qmpSockets := []govmmQemu.QMPSocket{
 		{
 			Type:   "unix",
 			Name:   q.qmpMonitorCh.path,
 			Server: true,
 			NoWait: true,
 		},
-	}, nil
+	}
+
+	if q.config.PVPanicEnabled || q.config.GuestMemoryDumpEnabled {
```

Comment:
I see that the PVPanicEnabled variable is being used only in conjunction with GuestMemoryDumpEnabled, which makes me wonder if we need PVPanicEnabled at all. As @jodh-intel mentioned, can we just get away with GuestMemoryDumpPath to enable/disable dumping memory?